docker/staging/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3.3'
+services:
+  coordinator:
+    build:
+      context: "../../"
+      dockerfile: "docker/Dockerfile"
+    networks:
+      - labgrid_net
+    tty: true
+    command: "crossbar start"
+  client:
+    build:
+      context: "../../"
+      dockerfile: "docker/Dockerfile"
+    networks:
+      - labgrid_net
+    tty: true
+    stdin_open: true
+    command: "/bin/bash"
+  exporter:
+    build:
+      context: "../../"
+      dockerfile: "docker/Dockerfile"
+    volumes:
+      - "./exporter-conf:/opt/exporter-conf"
+    devices:
+      - "/dev/ttyUSB0:/dev/ttyUSB0"
+    networks:
+      - labgrid_net
+    command: /opt/exporter-conf/entrypoint.sh
```

Comment:
As I see it, staging is an example setup that can be used for inspiration of deployment and sanity test. Hence I don't aim to make i to generic, moreover I think the entrypoint.sh is tightly coupled to the conf file. If one want to create custom he/she most properly have to change all three files

Reply:
True, the current entrypoint.sh is currently coupled with the configuration. So it is fair to keep it there (for now).
Looking forward, I think it would make sense to make this image generic so that it can be used together with custom configuration files mounted to a config path in the image.